app/views/admin/memberships/new.html.haml

```diff
@@ -1,13 +1,19 @@
-%table
-  %thead
-    %tr
-      %th Supplier
-      %th Actions
-  %tbody
-    - @suppliers.each do |supplier|
-      %tr
-        %td= supplier.name
-        %td
-          = form_for [:admin, @user, Membership.new] do |form|
-            = form.hidden_field :supplier_id, value: supplier.id
-            = form.submit 'Link user'
+.govuk-grid-row
+  .govuk-grid-column-full
+    %fieldset.govuk-fieldset
+      %legend.govuk-fieldset__legend.govuk-fieldset__legend--xl
+        %h1.govuk-fieldset__heading
+          Link to a supplier
+    %table.govuk-table
+      %thead.govuk-table__head
+        %tr.govuk-table__row
+          %th.govuk-table__header Supplier
+          %th.govuk-table__header Actions
+      %tbody.govuk-table__body
+        - @suppliers.each do |supplier|
+          %tr.govuk-table__row
+            %td.govuk-table__cell= supplier.name
+            %td.govuk-table__cell
+              = form_for [:admin, @user, Membership.new] do |form|
+                = form.hidden_field :supplier_id, value: supplier.id
+                = form.submit 'Link user', class: 'govuk-button govuk-!-margin-bottom-0'
```

Comment:
This should be a plain link as per the designs, a big table full of buttons may not be too clear for users?